src/Umbraco.Infrastructure/Examine/ContentValueSetBuilder.cs

```diff
@@ -86,6 +112,8 @@ public override IEnumerable<ValueSet> GetValueSets(params IContent[] content)
 
     private IEnumerable<ValueSet> GetValueSetsEnumerable(IContent[] content, Dictionary<int, IProfile> creatorIds, Dictionary<int, IProfile> writerIds)
     {
+        IDictionary<Guid, IContentType> contentTypeDictionary = _contentTypeService.GetAll().ToDictionary(x => x.Key);
```

Comment:
I am so suprised there is no better of doing it 😱 we retrieve a lot just to get keys.